cmd/http/config.json

```diff
@@ -1,28 +1,26 @@
 {
     "ConsistentHash": "rendezvous",
     "BucketSize": 10,
-    "ReplicaNum": 2,
+    "ReplicaNum": 3,
     "StoreType": "redis",
     "Concurrent": true,
     "Stores": [
         {
+            "RegionType": "local",
             "Name": "store1",
             "Host": "127.0.0.1",
             "Port": 6379
         },
         {
-            "Name": "store2",
-            "Host": "127.0.0.1",
-            "Port": 6380
-        },
-        {
+            "RegionType": "neighbor",
             "Name": "store3",
-            "Host": "172.31.12.96",
+            "Host": "172.31.9.140",
             "Port": 6379
         },
         {
+            "RegionType": "remote",
             "Name": "store4",
-            "Host": "172.31.12.96",
+            "Host": "172.31.9.140",
```

Comment:
not real code issue: consider to use a different ip address than the same one of neighbor to avoid potential confusion

Reply:
I have updated to avoid confusion.